src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -266,12 +266,22 @@ protected void customizeCellRenderer(JTable table, @Nullable Object value, boole
 
 
   public static class ContainsTextFilter implements EntryFilter {
```

Comment:
Since this is more than just matching containing text, perhaps rename it to something more general.  Maybe `LogEntryFilter`?  Or perhaps do away with the interface altogether and just have an `EntryFilter` class?  (I was thinking there'd be a separate implementation for each matching strategy but what you've go works great too. 👍

Reply:
(This is not a blocking ask.  Feel free to ignore and we can do something on another pass.)

Reply:
I choose a solution to merge them all in class `EntryFilter`.
If in the future we need more tyle filter -> we will refactor later.